src/rpc/lua.cc

```diff
@@ -46,6 +48,18 @@ LuaEngine::set_package_preload() {
   lua_pop(l_state, 2);
 }
 
+void
+LuaEngine::override_package_path() {
+  auto l_state = m_luaState;
+  lua_getglobal(l_state, "package");
+  lua_getfield(l_state, -1, "path");
+  std::string current_path(lua_tostring(l_state, -1));
+  std::string new_path = local_path + ';' + current_path;
+  lua_pushstring(l_state, new_path.c_str());
```

Comment:
Add empty newlines between lua and std::strings so it isn't so squished together.